src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_And_Leak.cs

```diff
@@ -146,25 +141,23 @@ public class Given_FrameworkElement_And_Leak
 #if __IOS__
 			LeakTestStyles.None // Disabled - #10344
 #elif __ANDROID__
-			LeakTestStyles.Default // Fluent styles disabled - #14340
+			LeakTestStyles.Uwp // Fluent styles disabled - #14340
```

Comment:
Intended to be `Fluent`?

Reply:
I think not here - the test is disabled for Fluent, previously Default == UWP, so this should be equivalent if I am not reading it wrong